src/http/ngx_http_wasm_filter_module.c

```diff
@@ -96,6 +101,11 @@ ngx_http_wasm_header_filter_handler(ngx_http_request_t *r)
 
     dd("ops resume rc: %ld", rc);
 
+    frc = ngx_http_wasm_flush_local_response(rctx);
```

Comment:
This invocation needs to go away; we should not give the false impression that calling `flush_local_response` twice is possible; in fact I have added an item on my TODO list to catch this from within the function some time soon. If the intention is to update the status code then we should update the status code, and not rely on this function to do it for us.

Reply:
The status code update could be a branch in the `send_local_response` host function of the Proxy-Wasm interface instead.